Mathlib/Analysis/SpecialFunctions/ExpDeriv.lean

```diff
@@ -62,7 +61,8 @@ theorem iter_deriv_exp : ∀ n : ℕ, (deriv^[n]) exp = exp
 #align complex.iter_deriv_exp Complex.iter_deriv_exp
 
 theorem contDiff_exp : ∀ {n}, ContDiff 𝕜 n exp := by
-  refine' contDiff_all_iff_nat.2 fun n => _
+  apply contDiff_all_iff_nat.2
+  intro n
```

Comment:
What was wrong with `refine`?

Reply:
It wasn't type checking, something to do with implicit lambdas.

I'm out of action for the next 12 hours, so please anyone feel free to make fixes / porting notes.

Reply:
I pushed a fix, it needed an `@`.